CONTRIBUTING.md

```diff
@@ -130,15 +126,15 @@ If you want to add a new component, we will need the following:
 
 ### Design
 
-Sketch is the preferred format for non-technical folks. We’d appreciate if you could provide an exact match of your component in Sketch format together with folders for each state e.g. disabled, expanded etc.
+Figma is the preferred format for non-technical folks. We’d appreciate if you could provide an exact match of your component in Figma format together with folders for each state e.g. disabled, expanded etc.
 
 ### Tokens
 
 Any visual CSS parameters of the component, such as *color, margins, paddings* etc. should not live as magic numbers in the component code, but as **tokens** in the [`@skyscanner/bpk-foundations-web`](https://github.com/Skyscanner/backpack-foundations/packages/bpk-foundations-web) package.
 
 Tokens are defined in the `src/base` directory (with the exception of product-specific tokens, which are in other subdirectories). Tokens come in two layers: In `aliases.json`, all base tokens are defined with concrete values, such as colours, numbers and sizes. The other files then map those aliases to tokens for specific elements.
 
-> You should probably not touch `aliases.json`, as our color palette or grid rarely changes.
+> You should probably not touch `aliases.json`, as our colour palette or grid rarely changes.
```

Comment:
![tenor-173838902](https://user-images.githubusercontent.com/30267516/128723744-d2a5bed4-67bb-424b-8e9b-7ff9cfd1a795.gif)

Reply:
miss you x